store/activity.go

```diff
@@ -174,7 +174,10 @@ func (s *Store) createActivityRaw(ctx context.Context, create *api.ActivityCreat
 		return nil, FormatError(err)
 	}
 
-	return activity, nil
+	if len(activityRawList) != 1 {
```

Comment:
If there are some conflicts, should we rollback instead of commit?

Reply:
Good idea, done.